tox.ini

```diff
@@ -11,16 +11,24 @@ deps =
     -rrequirements-test.txt
 whitelist_externals =
     make
+    mkdir
+    rm
 passenv = TOX_ENV_DIR
 setenv =
 	CACHITO_TESTING=true
+    PROMETHEUS_MULTIPROC_TEMP_DIR={envtmpdir}/prometheus_metrics
```

Comment:
This should probably be `PROMETHEUS_MULTIPROC_DIR`. And the indentation does not seem to match.

Reply:
Yes, with the other changes it does have to change.  Indents was a tabs vs spaces, fixed.